src/pages/de/core-concepts/framework-components.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Framework Komponenten
+description: Lerne wie du React, Svelte, etc. nutzen kannst.
+i18nReady: true
+---
+Erstelle deine Astro Webseite ohne dein favorisiertes Komponenten-Framework aufzugeben.
+
+Astro unterstützt eine Vielzahl von beliebten Frameworks wie [React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) und [Lit](https://lit.dev/).
+
+## Integrationen installieren
+
+Astro kommt mit optionalen Integrationen für React, Preact, Svelte, Vue, SolidJS, AlpineJS und Lit. Ein oder mehrere dieser Astro-Integrationen kann in deinem Projekt installiert und konfiguriert werden.
+
+Um Astro für die Verwendung dieser Frameworks zu konfigurieren, installiere zunächst die Integration mit ihren Abhängigkeiten:
+
+```bash
+npm install --save-dev @astrojs/react react react-dom
+```
+
+Anschließend importiere die Integration und füge die Funktion in deine Liste von Integrationen in der `astro.config.mjs` hinzu:
+
+```js title="astro.config.mjs" ins={3} ins=/(?<!p)react\\(\\)/
+import { defineConfig } from 'astro/config';
+
+import react from '@astrojs/react';
+import preact from '@astrojs/preact';
+import svelte from '@astrojs/svelte';
+import vue from '@astrojs/vue';
+import solid from '@astrojs/solid-js';
+import lit from '@astrojs/lit';
+import alpine from '@astrojs/alpinejs';
+
+export default defineConfig({
+	integrations: [react(), preact(), svelte(), vue(), solid(), lit(), alpine()],
+});
+```
+
+⚙️ Schaue den [Integrationsleitfaden](/en/guides/integrations-guide/) für mehr Details an und erfahre mehr über das Installieren und Konfigurieren von Astro-Integrationen.
```

Comment:
```suggestion
⚙️ Sieh dir den [Integrationsleitfaden](/de/guides/integrations-guide/) an, um mehr Details zum Installieren und Konfigurieren von Astro-Integrationen zu erfahren.
```

Fixing the English link reported by Link Checker, and replacing the too casual "schauen" with "sehen".